app/renderer/js/pages/preference/general-section.ts

```diff
@@ -493,4 +499,59 @@ export default class GeneralSection extends BaseSection {
 			await this.factoryResetSettings();
 		});
 	}
+
+	initSpellChecker(): void {
+		// The elctron API is a no-op on macOS and macOS default spellchecker is used.
+		if (process.platform === 'darwin') {
+			const note = document.querySelector('#note');
+			note.innerHTML += 'On macOS, the OS spellchecker is used.<br>Change the language from System Preferences -> Keyboard -> Text -> Spelling.';
```

Comment:
This is invalid HTML (due to the `->`s). We should always try to avoid HTML generation in favor of DOM manipulation:

```ts
note.append('On macOS, the OS spellchecker is used.');
note.append(document.createElement('br'));
note.append('Change the language from System Preferences -> Keyboard -> Text -> Spelling.');
```

Additionally, the UI strings should be internationalized.

(And although it doesn’t change the fundamental point, consider true Unicode arrows: →.)